soccer/KickEvaluator.cpp

```diff
@@ -355,24 +355,24 @@ void KickEvaluator::init_gradient_configs(
 
     // <PrevStart, Start>
     vector<tuple<float, float> > xStarts;
-    xStarts.reserve(robotStDevs->size());
+    xStarts.reserve(robotStDevs.size());
 
     // Add left boundary
     float startX = boundaryLower + *start_x_offset * *kick_std_dev;
-    xStarts.push_back(forward_as_tuple(boundaryLower, startX));
+    xStarts.emplace_back(forward_as_tuple(boundaryLower, startX));
```

Comment:
Since forward_as_tuple creates a tuple. You won't really get speedup from emplace. I think emplace_back(boundaryLower, startX) should foward correctly and cosntruct the tuple once. Or everything might be optimized to be the same anyways. Not sure. without the forward_as_tuple shouldn't be slower though.